src/main/java/com/hotels/beeju/core/BeejuCore.java

```diff
@@ -133,6 +162,33 @@ public String connectionURL() {
     return connectionURL;
   }
 
+  public Path tempDir() {
+    return tempDir;
+  }
+
+  public void deleteTempDir() {
+    try {
+      Files.walkFileTree(tempDir, new SimpleFileVisitor<Path>() {
```

Comment:
I've added this instead which is much neater, however this still needs to handle an `IOException` but the base class that we are extending `BeejuJUnitRule` from doesn't let you throw exceptions in the signature so it has to be caught in a try/catch - is there a better method of handling it than printing the stack trace?

Reply:
I think this should still just throw cause it can't handle it. In the rule it can `try catch(){throw new runtimeException(e);}` and I think in the extension where it is called it will just be thrown no need to do anything there.